README.md

```diff
@@ -0,0 +1,212 @@
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="splash.gif" alt="Budget App" width="300"  height="auto" />
+  <br/>
+
+  <h1><b>💰 Budget App</b></h1>
+
+  <p>Manage your Transactions in app</p>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About BudgetApp](#about-budget-app)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run Tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔮 Improvements](#improvements)
+- [🤝 Future Features](#future-features)
+- [⭐️ Show Your Support](#show-your-support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- ABOUT RECIPE DAILY -->
+
+# 📖 About BudgetApp <a name="about-budget-app"></a>
+
+Budget App is a web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Backend</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Frontend</summary>
+  <ul>
+    <li><a href="https://getbootstrap.com/">Bootstrap</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Flavorful Authentication:** Log in and register with a touch of Devise magic.
+- **Category** Create, edit, and showcase categories 
+- **Display Transactions** a list of transactionsfor each category. 
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+Start Budgeting!! [Not live yet](#).
```

Comment:
- [ ] Your live demo is also missing and this is an important requirement for this project please kindly add it before resubmit :+1:

Reply:
I have added the live demo